### PR TITLE
avple.tv

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/141263

### Add your comment and screenshots

None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: ChineseFilter/sections/adservers.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ! Bad: ||adssubdomain.legitwebsite.com^ (should be in adservers_firstparty.txt)
 !
 !
+||csjkb.com^
```
Comment (zloyden) [inline ChineseFilter/sections/adservers.txt:10]: Could you show this request via screenshot?

**Review by zloyden — COMMENTED** [review]
~~Seems this site is not Chinese. Could you move rules to AdGuard Japanese in the same files?~~

File: ChineseFilter/sections/adservers.txt (hunk 1 above)
Comment (piquark6046) [reply ChineseFilter/sections/adservers.txt:10]: I will delete because `||asserts.avple.tv/file/avple-asserts/ad.js` blocked a related content.

**Comment by piquark6046** [comment]
`FC2PPV-2933984` existing in the title is code of a pornographic film.

**Comment by zloyden** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/141666#issuecomment-1411941023 yep, I agree with it. It's just a title.

**Review by zloyden — APPROVED** [review]

**Pull request merged**